app/web/features/communities/CommunityPage/PlaceCard.tsx

```diff
@@ -3,41 +3,41 @@ import {
   CardActionArea,
   CardContent,
   CardMedia,
+  styled,
   useMediaQuery,
-  useTheme,
 } from "@mui/material";
 import Link from "next/link";
 import { Page } from "proto/pages_pb";
 import React, { useMemo } from "react";
 import LinesEllipsis from "react-lines-ellipsis";
 import { routeToPlace } from "routes";
-import makeStyles from "utils/makeStyles";
+import { theme } from "theme";
 import stripMarkdown from "utils/stripMarkdown";
```

Comment:
Please sanity check me but you might be able to just delete this component and its parent: `PlacesSection`. 

It seems this is only used in `PlacesSection` and that is not used anywhere.